### PR TITLE
Issue 4575 ast get docstring return type

### LINKED ISSUE #4575: ast.get_docstring: wrong return type
At least in recent Python versions it can return `None`: https://github.com/python/cpython/blob/0705ec8a149e27023b0420ae0366d16255f6c9f7/Lib/ast.py#L270-L293

`typed_ast` also seems to be affected, but has `Optional[bytes]` already:

```
stdlib/3/ast.pyi
194:def get_docstring(node: AST, clean: bool = ...) -> str: ...

third_party/3/typed_ast/ast27.pyi
15:def get_docstring(node: AST, clean: bool = ...) -> Optional[bytes]: ...

third_party/3/typed_ast/ast3.pyi
15:def get_docstring(node: AST, clean: bool = ...) -> str: ...

stdlib/2/ast.pyi
16:def get_docstring(node: AST, clean: bool = ...) -> str: ...
```
(@ 675ab77)

### PR DESCRIPTION
**Pull request by mitch292**
Resolves #4575

`get_docstring` can return `None` in >= 3.6.

cc: @blueyed & @JelleZijlstra

### CODE REVIEW

**Review by hauntsaninja — CHANGES_REQUESTED** [review]
Thanks! You'll want to address the CI lint failures.

Also the version check is incorrect, Python 3.5 returns None sometimes as well (it's just not as visible because the 3.5 source doesn't seem to be PEP8-compliant in this case). But you can demonstrate this empirically as well:
```
~ λ python3.5                                
Python 3.5.9 (default, Jun 11 2020, 19:03:06) 
[GCC 4.2.1 Compatible Apple LLVM 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ast
>>> mod = ast.parse("def f(): ...")
>>> mod.body
[<_ast.FunctionDef object at 0x1076dd080>]
>>> mod.body[0]
<_ast.FunctionDef object at 0x1076dd080>
>>> ast.get_docstring(mod.body[0])
>>> ast.get_docstring(mod.body[0]) is None
True
```

**Review by hauntsaninja — CHANGES_REQUESTED** [review]
Please remove the version check :-) It's good the way you had it on your first commit

File: third_party/3/typed_ast/ast3.pyi [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import sys
```
Comment (hauntsaninja) [inline third_party/3/typed_ast/ast3.pyi:1]: One more nit: unused import

**Comment by hauntsaninja** [comment]
Thanks again!

**Comment by mitch292** [comment]
Thanks for the help @hauntsaninja !

**Pull request merged**